ai_presenter/ai_presenter.py

```diff
@@ -19,19 +19,26 @@ def run(self):
                 f' {actor.gender}, {actor.description}. '
         textai.send(message)
 
+        with open('text_ai.txt', 'w') as file:
+            textai = self.generator.get_text()
+            for key, scene in self.database.scenes.items():
+                logging.info(f"******** \nWorking on scene: {scene.name} in " +
+                             f"{scene.location}")
+                output = AIPresenter.generate_ai(self, scene, textai)
+                file.write(output + '\n')
+                logging.info(f'got back from textai: {output}')
+
+    def generate_ai(self, scene, textai):
```

Comment:
change method from `generate_ai` to something more specific to indicate that this is getting the text ai output

Reply:
Should also probably specify that scene be of type Scene with `scene: Scene` and textai be of type TextAI with `textai: TextAI`